Source/WebCore/platform/graphics/texmap/TextureMapperLayer.h

```diff
@@ -187,14 +187,14 @@ class TextureMapperLayer : public CanMakeWeakPtr<TextureMapperLayer> {
     void collect3DRenderingContextLayers(Vector<TextureMapperLayer*>&);
 
 #if ENABLE(DAMAGE_TRACKING)
-    bool canInferDamage() const { return m_damagePropagation && !m_damage.isInvalid(); }
+    bool canInferDamage() const { return m_damagePropagation && !m_receivedDamage.isInvalid(); }
     void collectDamageRecursive(TextureMapperPaintOptions&, Damage&);
     void collectDamageSelfAndChildren(TextureMapperPaintOptions&, Damage&);
     void collectDamageSelf(TextureMapperPaintOptions&, Damage&);
     void collectDamageSelfChildrenReplicaFilterAndMask(TextureMapperPaintOptions&, Damage&);
     void collectDamageSelfChildrenFilterAndMask(TextureMapperPaintOptions&, Damage&);
     void damageWholeLayerDueToTransformChange(const TransformationMatrix& beforeChange, const TransformationMatrix& afterChange);
-    FloatRect transformRectForDamage(const FloatRect&, const TransformationMatrix&, const TextureMapperPaintOptions&);
+    FloatRect transformRectFromLayerToGlobalCoordinateSpace(const FloatRect&, const TransformationMatrix&, const TextureMapperPaintOptions&);
```

Comment:
It looks to me that this could be marked `const`, doesn't seem to change the state of the class

Reply:
I made it static-local eventually